src/routes/api-documentation.ts

```diff
@@ -0,0 +1,44 @@
+import Router from '@koa/router';
+import { Context, Middleware, Next } from 'koa';
+import { hydra, owl, schema } from 'rdf-namespaces';
+import Routes from './index';
+
+export default (router: Router): Middleware => (
+  async ({ request, response }: Context, next: Next): Promise<void> => {
+    response.body = {
+      '@context': {
+        '@base': request.origin,
+      },
+      '@id': router.url(Routes.ApiDocumentation),
+      '@type': hydra.ApiDocumentation,
+      [hydra.entrypoint]: { '@id': router.url(Routes.EntryPoint) },
+      [hydra.supportedClass]: [
+        {
+          '@id': schema.EntryPoint,
+          '@type': hydra.Class,
+          [hydra.title]: { '@value': 'API entry point', '@language': 'en' },
+          [hydra.supportedOperation]: {
```

Comment:
Providing this supported operation is an overkill, I think. Providing a `hydra:entrypoint` predicate is enough as it is a `hydra:Link`.